components/list/list-item-mixin.js

```diff
@@ -118,6 +119,11 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 				margin-left: 0.9rem;
 				margin-right: 0;
 			}
+			[slot="content"] ::slotted(d2l-icon[slot="illustration"]),
+			[slot="content"] .d2l-list-item-illustration d2l-icon {
+				border-radius: 0;
```

Comment:
We were talking about the `border-radius` amongst the team earlier today, and it sounds like in hindsight we would have preferred the list-item wasn't so opinionated about this property, and that we'd rather let consumers always control their `border-radius`. I think we're just going to run into more cases were it needs to be overridden, and not necessarily be an icon. We'd like to look into the option of perhaps changing this... but obviously existing usages of the component would need to start specifying it if we remove it.

Reply:
Right. I was having a hard time thinking of other things people would put in here that _wouldn't_ want the radius, and I figured `d2l-icon` would be a fairly common one. @m6jones had suggested a separate `slot` for icons that maybe could be more generic and not have the radius and that wouldn't disrupt existing usages.

Reply:
Yeah it's hard to know what they want.  I think for now, we add this rule to unblock you, and we'll start going through the existing usages to apply border radius so we can remove this in the near future without regressions.